modules/react/radio/lib/RadioGroup.tsx

```diff
@@ -39,9 +39,14 @@ const Container = styled('div')<Pick<RadioGroupProps, 'error' | 'grow' | 'theme'
     boxSizing: 'border-box',
     borderRadius: borderRadius.m,
     padding: `${space.xxxs} ${space.xs}`,
-    margin: `-${space.xxxs} -${space.xs}`,
+    margin: `calc(-${space.xxxs} + 2px) -${space.xs}`,
     '& > div': {
       margin: `${space.xxs} ${space.zero}`,
+      alignItems: 'flex-start',
+      '> div': {
+        flex: '0 0 auto',
+        marginTop: '1px', // visually align radio button down
```

Comment:
Flex start seems to move the radio buttons up, so I added this margin top to visually align everything based on the chromatic diff.